eigrpd/eigrp_cli.c

```diff
@@ -79,7 +79,7 @@ DEFPY_YANG(
 		 as_str, vrf ? vrf : VRF_DEFAULT_NAME);
 
 	nb_cli_enqueue_change(vty, xpath, NB_OP_DESTROY, NULL);
-	return nb_cli_apply_changes(vty, NULL);
+	return nb_cli_apply_changes_clear_pending(vty, NULL);
```

Comment:
Just wondering, do all `no router` commands really have to be executed without batching? Or this is just a precaution?

Reply:
It's precaution. Who knows what bugs lie in wait, or haven't been written yet. Executing the batched commands on each no seems reasonable for the time being until we fully convert everything (and get rid of those bgp peer-group style bugs).